pythran/transformations/expand_globals.py

```diff
@@ -46,6 +48,10 @@ def visit_Module(self, node):
                 continue
             for target in stmt.targets:
                 assert isinstance(target, ast.Name)
+                if target.id in self.to_expand:
+                    raise PythranSyntaxError(
```

Comment:
That's already done. in fact the code already rasied an error, but
blaming a function redefinition as globals are turned into functions.